docs/terminal/melbourne.md

```diff
@@ -203,98 +203,105 @@ The Standard assignable level from ENR to ML TCU is `A090`. All other levels mus
 
 ### ML TCU / ML ADC
 #### Auto Release
+!!! important
+    Melbourne utilises auto release for all **Procedural** SIDs and the **ML (RADAR)** SID provided aircraft are assigned the Standard Assignable Level and a [Standard Assignable Heading](#standard-assignable-departure-headings).
 
-Auto Release shall be used for aircraft that are:      
-a) Departing from a runway nominated on the ATIS; and  
-b) Issued a Procedural SID, or the ML (RADAR) SID with a [Standard assignable heading](#standard-assignable-headings); and   
-c) Assigned `A050`.
+'Next' coordination is **not** required for aircraft that are:   
+    a) Departing from a runway nominated on the ATIS; and  
+    b) Assigned `A050`; and  
+    c) Assigned a **Procedural** SID; or  
+    d) Assigned a [Standard Assignable Heading](#standard-assignable-departure-headings)
 
 Any aircraft that don't meet these criteria must be coordinated to ML TCU with a "Next" Call.  
 
-"Next" Coordination is a procedure where the ML ADC controller gives a heads-up to the ML TCU controller about an impending departure. The ML TCU controller will respond by assigning any relevant departure instructions (i.e. assigned heading, amended altitude, visual turn) to the aircraft, for the ML ADC controller to pass on with their takeoff clearance.
-
 !!! example
     <span class="hotline">**ML ADC** -> **ML TCU**</span>: "Next, CYF, runway 34"  
     <span class="hotline">**ML TCU** -> **ML ADC**</span>: "CYF, track extended centreline, unrestricted"  
-    <span class="hotline">**ML ADC** -> **ML TCU**</span>: "Track extended centreline, unrestricted, CYF"
+    <span class="hotline">**ML ADC** -> **ML TCU**</span>: "Track extended centreline, CYF"
 
-##### Standard Assignable Headings
+##### Standard Assignable Departure Headings
 The following Standard Assignable Headings may be used for aircraft assigned the ML (RADAR) SID, depending on their direction of travel.
 
 | Runway | W | NE | SE | S |
 | ---- | ---- | ---- | ---- | ---- |
-| 16* | 290 | 290 | 160 | 260 |
+| 09* | - | - | - | - |
+| 16** | 290 | 290 | 160 | 260 |
 | 27 | 320 | 290 | 260 | 260 |
 | 34 | 340 | 340 | 340 | 260 |
 
+*When Runway 09 is in use for departures, ML TCU shall nominate a heading to ML ADC for use as a standard assignable heading.
 
-*\*When Runway 09 is in use for arrivals, all Runway 16 departures shall be assigned heading 160.*
-
-When Runway 09 is in use for departures, ML TCU shall nominate a heading to ML ADC for use as a standard assignable heading.
+**When Runway 09 is in use for arrivals, all Runway 16 departures shall be assigned heading 160.
 
 ### ML TCU / EN ADC
+
+#### Start Clearance
+When an aircraft requests start clearance, the EN SMC controller shall coordinate with ML TCU to obtain the start clearance.
```

Comment:
Does this apply to all departures, or just departures into TCU class C?

Reply:
this was already in beforehand, but basically if the aircraft requests start clearance, then EN SMC needs to ask ML TCU for the clearance. ERSA outlines the requirements for requesting start clearance (not that most pilots on vatsim would be following that anyway)

Reply:
Should we change this to only aircraft who will enter the TCU? I figure if an aircraft wants to exit the coffin into class G or remain in EN airspace, it doesn't make sense to bother ML with it

Reply:
We could. I figure it was worded as such to account for the fact that pilots often won't follow the ERSA and request start clearance. It's more of a pilot-initiated coord than it is sector-initiated coord